### PR TITLE
VTCCA-0 resolving qa issues due to broke endpoint

### PR DESCRIPTION
**Pull request by DanielConnelly**

### CODE REVIEW

**Comment by darrell-plant** [comment]
LGTM 👍

File: app/controllers/agentAppointment/AppointAgentController.scala [hunk 1]
```diff
@@ -75,11 +75,10 @@ class AppointAgentController @Inject()(
       .bindFromRequest()
       .fold(
         hasErrors = errors => {
-          agentsConnector.ownerAgents(request.organisationId) map { ownerAgents =>
+          getAgents(request.organisationId) map { ownerAgents =>
```
Comment (PeterPerhac) [inline app/controllers/agentAppointment/AppointAgentController.scala]: i think we have agreed to not use infix notation for `map` `flatMap` etc. in production code. It's ok in tests to say `foo shouldBe bar` and not `foo.shouldBe(bar)` but in code we'd like to always consistently use the dot `foo.map(_ => true)`

File: app/controllers/agentAppointment/AppointAgentController.scala [hunk 2]
```diff
@@ -94,11 +93,11 @@ class AppointAgentController @Inject()(
                     None)))
             case None =>
               val errors: List[FormError] = List(invalidAgentCode)
-              agentsConnector.ownerAgents(request.organisationId) flatMap { ownerAgents =>
+              getAgents(request.organisationId) flatMap { ownerAgents =>
```
Comment (PeterPerhac) [inline app/controllers/agentAppointment/AppointAgentController.scala]: i think we have agreed to not use infix notation for `map` `flatMap` etc. in production code. It's ok in tests to say `foo shouldBe bar` and not `foo.shouldBe(bar)` but in code we'd like to always consistently use the dot `foo.map(_ => true)`

**Comment by PeterPerhac** [comment]
Cool, Thanks Dan! 👍 I know my comments were on someone else's original code, but I now noticed. It's up to you if you want to address the minor nitpicks.

**Pull request merged**